components/pkg-export-tar/src/lib.rs

```diff
@@ -53,17 +53,17 @@ pub fn export_for_cli_matches(ui: &mut UI, matches: &clap::ArgMatches) -> Result
 pub fn export(ui: &mut UI, build_spec: BuildSpec) -> Result<()> {
     let build_result = build_spec.create(ui).unwrap();
     let builder_dir_path = build_result.0.path();
-    let builder_idents = build_result.1;
+    let pkg_ident = build_result.1;
 
-    tar_command(builder_dir_path, builder_idents);
+    tar_command(builder_dir_path, pkg_ident);
     Ok(())
 }
 
 #[allow(unused_must_use)]
-fn tar_command(temp_dir_path: &Path, builder_idents: Vec<PackageIdent>) {
+fn tar_command(temp_dir_path: &Path, pkg_ident: PackageIdent) {
     // We currently support only exporting one tar package at a time
     // So we grab the first value from the Vec of idents
-    let pkg_ident = builder_idents[0].clone();
+    let pkg_ident = pkg_ident.clone();
```

Comment:
I don't think you need this line at all anymore.

Also, the comment immediately above no longer applies.